app/code/Magento/Catalog/Test/Unit/Controller/Category/ViewTest.php

```diff
@@ -37,168 +37,177 @@
 
-        $this->resultFactory = $this->createMock(ResultFactory::class);
-        $this->resultFactory->expects($this->any())->method('create')->willReturn($this->page);
+        $this->resultFactoryMock = $this->createMock(ResultFactory::class);
+        $this->resultFactoryMock->method('create')->willReturn($this->pageMock);
 
-        $this->context = $this->createMock(Context::class);
-        $this->context->expects($this->any())->method('getRequest')->willReturn($this->request);
-        $this->context->expects($this->any())->method('getResponse')->willReturn($this->response);
-        $this->context->expects($this->any())->method('getObjectManager')
-            ->willReturn($this->objectManager);
-        $this->context->expects($this->any())->method('getEventManager')->willReturn($this->eventManager);
-        $this->context->expects($this->any())->method('getView')->willReturn($this->view);
-        $this->context->expects($this->any())->method('getResultFactory')
-            ->willReturn($this->resultFactory);
+        $this->contextMock = $this->createMock(Context::class);
+        $this->contextMock->method('getRequest')->willReturn($this->requestMock);
+        $this->contextMock->method('getResponse')->willReturn($this->responseMock);
+        $this->contextMock->method('getObjectManager')
+            ->willReturn($this->objectManagerMock);
+        $this->contextMock->method('getEventManager')->willReturn($this->eventManagerMock);
+        $this->contextMock->method('getView')->willReturn($this->viewMock);
+        $this->contextMock->method('getResultFactory')
+            ->willReturn($this->resultFactoryMock);
 
-        $this->category = $this->createMock(\Magento\Catalog\Model\Category::class);
-        $this->categoryRepository = $this->getMockForAbstractClass(CategoryRepositoryInterface::class);
+        $this->categoryMock = $this->createMock(\Magento\Catalog\Model\Category::class);
+        $this->categoryRepositoryMock = $this->getMockForAbstractClass(CategoryRepositoryInterface::class);
 
-        $this->store = $this->createMock(Store::class);
-        $this->storeManager = $this->getMockForAbstractClass(StoreManagerInterface::class);
-        $this->storeManager->expects($this->any())->method('getStore')->willReturn($this->store);
+        $this->storeMock = $this->createMock(Store::class);
+        $this->storeManagerMock = $this->getMockForAbstractClass(StoreManagerInterface::class);
+        $this->storeManagerMock->method('getStore')->willReturn($this->storeMock);
 
-        $this->catalogDesign = $this->createMock(Design::class);
+        $this->catalogDesignMock = $this->createMock(Design::class);
 
         $resultPageFactory = $this->getMockBuilder(PageFactory::class)
             ->disableOriginalConstructor()
             ->setMethods(['create'])
             ->getMock();
-        $resultPageFactory->expects($this->atLeastOnce())
+        $resultPageFactory->expects(self::atLeastOnce())
```

Comment:
```suggestion
        $resultPageFactory->expects($this->atLeastOnce())
```

Could you replace back static call to the calling from this, just for the consistency across the platform?
please do the same for all other places

Reply:
Done @ihor-sviziev